pkg/api/v1/metric_usage.go

```diff
@@ -111,10 +111,16 @@ type RuleUsage struct {
 	Expression string `json:"expression"`
 }
 
+type DashboardUsage struct {
+	UID  string `json:"uid"`
```

Comment:
UID or ID ? Big question actually since in Perses there is no UID but an unique ID which is a combination of different field in the `metadata` struct

Reply:
I think ID is more accurated in this context, wdyt?

Reply:
sounds good !